vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -1090,6 +1090,7 @@ def __init__(
         zero_expert_type: str | None = None,
         expert_mapping: list[tuple[str, str, int, str]] | None = None,
         n_shared_experts: int | None = None,
+        routing_method_type: int = 2,
```

Comment:
This should use the enum and probably be None by default so it can be derived. It certainly doesn't make sense to default to DeepSeek routing aka 2